src/main/java/frc/robot/commands/ShooterOnBlue.java

```diff
@@ -53,6 +23,8 @@ public void initialize() {
     // Called every time the scheduler runs while the command is scheduled.
     @Override
     public void execute() {
+        m_shooter.runTopMotor(blue_top);

```

Comment:
Not a huge deal, but unless there is a good reason to change it, I would suggest using "setTopMotor" since that is more in alignment with the underlying verbs associated with the controller.